books/scss/styles.scss

```diff
@@ -1,5 +1,9 @@
 body {
+  background-color: $wild-sand;
+  display: flex;
   font-family: $roboto;
+  height: 100vh;
```

Comment:
This won't be the case with every view. Are you sure you need this?

Reply:
It has been enhanced to cover getting more views. I use this to get the full height of the screen and don't burn sizes in px.